test/jdk/java/foreign/NativeTestHelper.java

```diff
@@ -38,19 +38,15 @@
     public static final SystemABI ABI = Foreign.getInstance().getSystemABI();
 
     public static boolean isIntegral(MemoryLayout layout) {
-        var optAbiType = layout.abiType();
-        if (!optAbiType.isPresent()) {
-            return false;
-        }
-        return switch(optAbiType.get()) {
+        return switch(SystemABI.Type.fromLayout(layout)) {
```

Comment:
Need to catch the exception?

Reply:
No, I don't think so. The intent is to have a hard failure if the ABI type is missing. The old code was failing more silently by returning `false`.

Reply:
Similar motivation here: https://github.com/openjdk/panama-foreign/pull/64#discussion_r398488164

Reply:
OK, I didn't look at use cases for this method, just thinking of compatibility of existing code. If the cases are expecting native ABI type to be available all the time, then yes, I agree.